service_quotas/ec2_availability.go

```diff
@@ -0,0 +1,58 @@
+package servicequotas
```

Comment:
I think it makes sense to move the `AvailableIpsPerSubnetUsageCheck` into the ec2_limits as it is still using the same API and the actual mapping to a service quota code happens elsewhere.